Can't build `playground` locally

I forked the prql repo to my own account, then cloned it to my machine (macOS 10.15.7). I then tried to build the playground so I could try it locally. I got an error (`sh: wasm-pack: command not found`), so I added `"wasm-pack": "^0.10.3"` to the devDependencies in the `package.json` file.

I then tried the following. I saw a couple warnings about deprecations, but I don't know what to do about the `cargo metadata` file. What other troubleshooting information could I provide? Many thanks.

```
√ playground % node --version
v16.14.0
√ playground % npm --version
8.3.1
√ playground % npm install
npm WARN deprecated stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
npm WARN deprecated svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.

> prql-playground@0.1.0 preinstall
> cd ../prql-js && npm run build-bundler


> prql-js@0.2.6 build-bundler
> wasm-pack build --target bundler --release --out-dir dist/bundler

Error: Error during execution of `cargo metadata`: error: failed to parse lock file at: /Users/richb/github/prql/Cargo.lock

To learn more, run the command again with --verbose.

npm ERR! code 1
npm ERR! path /Users/richb/github/prql/playground
npm ERR! command failed
npm ERR! command sh -c cd ../prql-js && npm run build-bundler

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/richb/.npm/_logs/2022-09-14T17_44_20_052Z-debug-0.log
?1 playground %
```

--- a/playground/README.md
+++ b/playground/README.md
@@ -1,10 +1,16 @@
 # PRQL Playground
 
-Uses `wasm-react-scripts` instead of `react-scripts` until <https://github.com/facebook/create-react-app/pull/8303> is merged.
+A fast-feedback compiler from PRQL to SQL, hosted at <https://prql-lang.org/playground/>
 
-Development:
+To run locally, [set up a development
+environment](../CONTRIBUTING.md#development-environment), and then run:
 
 ```sh
 npm install
 npm start
 ```
+
+## Notes
+
+This currently uses `wasm-react-scripts` instead of `react-scripts`, until
+<https://github.com/facebook/create-react-app/pull/8303> is merged.

